### DISCUSSION TITLE
The "Set WLAN Hotspot" function of the system cannot be opened after using the software

### DISCUSSION THREAD

**Discussion by NianBroken** [Mygod/VPNHotspot#336] [Q&A]
After modifying the parameters of "WLAN Hotspot".
The "Set WLAN Hotspot" function that comes with the system can no longer be opened.
The system's "Set WLAN Hotspot" flashes back when you open it.

My phone model: Xiaomi Mi 10
System version: MIUI 13.0.2
Android version: 12

**Comment by Mygod** [comment]
Report to MIUI instead.
